src/storage/projected_columns.cpp

```diff
@@ -5,6 +5,17 @@
 #include <utility>
 #include <vector>
 namespace terrier::storage {
+const uint32_t ProjectedColumns::AttrSizeForColumn(uint16_t col_id) {
```

Comment:
attr_ends_ seems to be only for getting attr sizes, so why not directly just store attr size? Or would attribute size potentially be too large to fit them all in the header?

Reply:
Ignore this, John you're a genius. It make sense